api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -1085,7 +1113,12 @@ public DbUser updateRasLinkLoginGovStatus(String loginGovUserName) {
   public DbUser confirmProfile(DbUser dbUser) {
     return updateUserWithRetries(
         user -> {
-          user.setProfileLastConfirmedTime(new Timestamp(clock.instant().toEpochMilli()));
+          Timestamp timestamp = new Timestamp(clock.instant().toEpochMilli());
+          // user.setProfileLastConfirmedTime() will be replaced by
+          // accessModuleService.updateCompletionTime()
+          user.setProfileLastConfirmedTime(timestamp);
```

Comment:
Profile and Publications are also set in createUser() so we'll need to add dual writes there too.  (Line 474)

Reply:
ah, nice catch! Thanks!